### ISSUE TITLE
deprecate auto-shimming if no longer needed

### ISSUE THREAD

**Issue by AlexAegis** [AlexAegis/pakk#5]
Once this is merged this feature might no longer be needed.
https://github.com/pnpm/cmd-shim/pull/43
https://github.com/pnpm/pnpm/issues/1801

Also check if this is needed with yarn. Keep if it is.
